content/tv/tanzu-tuesdays/0067.md

```diff
@@ -0,0 +1,34 @@
+---
+Date: '2021-08-31T12:00:00-07:00'
+Description: "Let's talk about SpringOne 2021!"
+PublishDate: '2020-01-04T00:00:00-07:00'
+aliases:
+- /tv/tanzu-tuesdays/67
+draft: false
+episode: '67'
+episode_banner: /images/tv/episodes/tanzu-tuesdays/0067.png
+episode_image: /images/tv/episodes/tanzu-tuesdays/0067.png
+explicit: 'no'
+guests:
+- Cora Iberkleid
+- Glenn Renfro
+- Ryan Baxter
+- Madhura Bhave
+- Joe Grandja
+- Spencer Gibb
+hosts:
+- Tiffany Jernigan
+- Whitney Lee
+- Leigh Capili
+images:
+- /images/tv/episodes/tanzu-tuesdays/0067.png
+lastmod: '2021-08-09'
+minutes: 60
+title: "Let's talk about SpringOne 2021!"
+truncate: ''
+twitch: vmwaretanzu
+youtube: ""
+type: tv-episode
+---
+
+Join us to hear about exciting upcoming talks for this week's [SpringOne](springone.io) conference!
```

Comment:
```suggestion
Join us to hear about exciting upcoming talks for this week's [SpringOne](https://springone.io) conference!
```

Reply:
Fixed, thanks

Reply:
@tiffanyfay looks like Hugo's renderer doesn't recognize the `.io` TLD for some reason, I'm assuming this should fix it